composer.json

```diff
@@ -1,32 +1,29 @@
 {
-	"name" : "hostnet/dead-tool",
-	"description" : "Dead Code Identification Toolbox",
-	"license" : "EPL-1.0",
-	"require" : {
-		"php" : ">=5.3",
-		"pear/console_commandline" : "*@dev",
-		"pear/pear_exception" : "*@dev"
-	},
-	"minimum-stability" : "stable",
-	"repositories" : [{
-			"type" : "composer",
-			"url" : "http://packagist.hostnetbv.nl"
-		}
-	],
-	"authors" : [{
-			"name" : "Hidde Boomsma",
-			"email" : "hboomsma@hostnet.nl"
-		}
-	],
-	"autoload" : {
-		"classmap" : [
-			"src/"
-		]
-	},
-	"bin" : [
-		"bin/dead"
-	],
-	"require-dev" : {
-		"phpunit/phpunit" : "3.7.*"
-	}
-}
\ No newline at end of file
+    "name":              "hostnet/dead-tool",
+    "description":       "Dead Code Identification Toolbox",
+    "license":           "MIT",
```

Comment:
License has changed, but there's no LICENSE file in the repository. 

You should be able to use this one (copied from hostnet/form-handler-bundle):
```
Copyright (c) 2014 Hostnetbv

 Permission is hereby granted, free of charge, to any person
 obtaining a copy of this software and associated documentation
 files (the "Software"), to deal in the Software without
 restriction, including without limitation the rights to use,
 copy, modify, merge, publish, distribute, sublicense, and/or sell
 copies of the Software, and to permit persons to whom the
 Software is furnished to do so, subject to the following
 conditions:

 The above copyright notice and this permission notice shall be
 included in all copies or substantial portions of the Software.

 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
 EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
 OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
 NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
 HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
 WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
OTHER DEALINGS IN THE SOFTWARE.
```